user-guide/tls-termination.md

```diff
@@ -14,7 +14,7 @@ This guide requires you have the following installed:
 
 ## Install Ambassador
 
-Install Ambassador in Kubernetes using either [YAML manifests](https://www.getambassador.io/user-guide/getting-started) or the [Helm chart](https://www.getambassador.io/user-guide/helm). If you do not know which to do, use the YAML manifests.
+Install Ambassador in Kubernetes using either [YAML manifests](../../user-guide/getting-started) or the [Helm chart](../../user-guide/helm). If you do not know which to do, use the YAML manifests.
```

Comment:
so since tls-termination is in user-guide already, shouldn't these be ../ and not ../../?

Reply:
I could have simplified it to `../getting-started` from `../../user-guide/getting-started` (and likewise for `helm`), but it doesn't really matter.